lib/Watchman.js

```diff
@@ -31,7 +31,8 @@ module.exports = class Watchman {
       },
       json: true, // Automatically parses the JSON string in the response
       resolveWithFullResponse: true,
-      simple: false
+      simple: false,
+      qs: { timeoutSeconds: 120 }
```

Comment:
want to read that from a configmap so we can easily override it when needed?

Reply:
yes. I wanted to make this PR just so i did forget the fix. Lets make sure we do that before this is merged.